1-js/02-first-steps/05-types/article.md

```diff
@@ -58,6 +58,31 @@ Kodunuzun tamamı hiç durmadan çalışacaktır. En kötü ihtimalle `NaN` sonu
 Özel sayısal değerler "number" tipine aittir. Tabiki sayı bizim bildiğimiz tipte sayı değillerdir. 
 <info:number> bölümünde sayısal değerler ile çalışmayı daha derinlemesine göreceksiniz.
 
+## BigInt - Büyük Sayı
+
+JavaScript'te "number" türü, şundan büyük tamsayı değerlerini temsil edemez <code>(2<sup>53</sup>-1)</code> (bu `9007199254740991`), veya daha az <code>-(2<sup>53</sup>-1)</code> negatifler için. Dahili temsillerinden kaynaklanan teknik bir sınırlamadır.
+
+Çoğu amaç için bu oldukça yeterlidir, ancak bazen gerçekten büyük sayılara ihtiyacımız olabilir, kriptografi veya mikrosaniye hassasiyetli zaman damgaları için.
+
+Son zamanlarda, isteğe bağlı uzunluktaki tam sayıları temsil etmek için dile `BigInt` türü eklendi.
+
+Bir tamsayının sonuna `n` eklenerek `BigInt` değeri oluşturulur:
+
+```js
+// Sondaki "n" bu değerin bir BigInt olduğu anlamına gelir
+const bigInt = 1234567890123456789012345678901234567890n;
+```
+
+`BigInt` sayılarına nadiren ihtiyaç duyulduğundan, onları burada ele almıyoruz, ancak onlara ayrı bir bölüm <info:bigint> ayırdık. Bu kadar büyük sayılara ihtiyacınız olduğunda okuyun.
+
+
+```smart header="Compatibility issues"
+Şu anda, `BigInt` Firefox/Chrome/Edge/Safari'de destekleniyor, ancak IE'de desteklenmiyor.
+```
+
+You can check [*MDN* BigInt compatibility table](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#Browser_compatibility) to know which versions of a browser are supported.
```

Comment:
looks like there is an english sentence here

Reply:
I edited